bin/oneoff/upload_non_en_videos.rb

```diff
@@ -0,0 +1,126 @@
+# This script is used to batch upload non-EN videos, with the assumption that
+# we will frequently get these in batches. They will be uploaded to YouTube
+# and S3 as well as added to videos.csv.
+
+require 'google/apis'
+require 'google/apis/youtube_v3'
+require 'google/api_client/client_secrets.rb'
+require 'googleauth'
+require 'googleauth/stores/file_token_store'
+require 'csv'
+require 'fileutils'
+gem 'google-api-client', '>0.7'
+require 'rubygems'
+require 'csv'
+
+require_relative '../../dashboard/config/environment'
+
+# The directory where the files are in. There can be multiple folders in this folder,
+# but those must be noted in the map file below
+VIDEO_FILE_DIRECTORY = ''
+# CSV that maps video keys to their title and filename
+# Must have columns titled 'Folder', 'Partner filename', 'Key', and 'Video Title'
+MAP_FILE = 'test.csv'
+# Switch this to false when ready to run
+DRY_RUN = true
+LOCALE = 'es-MX'
+
+APPLICATION_NAME = 'YouTube Data API Ruby Tests'
+
+# REPLACE WITH NAME/LOCATION OF YOUR client_secrets.json FILE
+CLIENT_SECRETS_PATH = ''
+
+# This is the URI so that we can authorize from a browser. Don't change this.
+REDIRECT_URI = 'urn:ietf:wg:oauth:2.0:oob'
```

Comment:
how was this value obtained?

Reply:
https://developers.google.com/identity/protocols/OAuth2InstalledApp